services/site/src/components/Navigation.tsx

```diff
@@ -35,78 +37,159 @@ const Navigation: React.FC<React.PropsWithChildren<NavigationProps>> = ({ displa
 
   return (
     <>
-      <header className={clsx("pt-8 pb-6 px-6 sm:px-8 grid grid-cols-2 relative z-20 bg-background", "sm:px-11", "lg:grid-cols-3")}>
+      <header
+        className={clsx(
+          "pt-2 pb-4 px-6 sm:px-8 grid grid-cols-12 z-20 bg-background",
+          "sm:px-11",
+          "md:pt-4",
+          "lg:pb-2 lg:pt-1",
+          isSticky && "fixed top-0 w-full",
+          !isSticky && "relative",
+        )}
+      >
         <Link
           href="/"
           className={clsx(
-            "block text-light border-none rounded col-span-1 max-w-max py-1 px-2 -ml-2",
+            "block text-light border-none rounded col-span-4 max-w-max py-1 px-2 -ml-2",
             "transition ease-in-out duration-300",
             "motion-safe:hover:scale-110 motion-reduce:border-solid",
             "focus-outline-offset",
           )}
         >
-          <A11yphantLogo className="w-48 xs:w-52 sm:w-36" />
+          <A11yphantLogo className="w-48 xs:w-52 md:w-36" />
           <span className="sr-only">Allyphant Homepage</span>
         </Link>
-        {displayBreadcrumbs && (
-          <div
-            className={clsx(
-              "mt-4 col-span-2 w-fit-content self-center",
-              "sm:hidden",
-              "lg:mt-0 lg:col-span-1 lg:block lg:ml-[50%] lg:-translate-x-1/2",
-            )}
-          >
-            <Breadcrumbs />
-          </div>
-        )}
-        <div className={clsx("flex justify-end items-center")}>
-          <div className={clsx("hidden justify-end items-center mr-4", "sm:flex")}>
-            {children}
-            {!currentUser?.isRegistered && (
-              <>
-                <Button
-                  primary
-                  onClick={() => {
-                    userAccountModalApi.show("signup");
-                  }}
-                  className="mx-4 px-6"
-                >
-                  Sign Up
-                </Button>{" "}
-                <Button
-                  onClick={() => {
-                    userAccountModalApi.show("login");
-                  }}
-                  className={clsx("px-8 py-3 border-none", "hover:border-primary-dark", "focus:bg-transparent")}
-                >
-                  Login
-                </Button>
-              </>
-            )}
-          </div>
-          {currentUser?.isRegistered && (
+
+        {/* Mobile Nav */}
+        <div className={clsx("flex justify-end items-center col-span-8", "md:hidden")}>
+          {children}
```

Comment:
We can remove children completely since 

1. we don't use it anymore
2. even before, the placement and purpose of the children was pretty random